content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -52,6 +52,7 @@ This is the path followed by DNS Queries after NodeLocal DNSCache is enabled:
 {{< /note >}}
 
 This feature can be enabled using the following steps:
+
```

Comment:
@prameshj, The `sed` commands are okay. I was wondering why the manifest required further configuration.
It would be nice to indent the variables, lines 59 - 65 under the second list item.

Reply:
I have indented lines 59-65, could you take a look again?
The manifest needs further config because some variables are different depending on the environment this is deployed in. There are some developer scripts that substitute the variables in some environments, this doc provides the background to indetify what to substitute.

Reply:
Thanks again, for the review @kbhawkey